variables.tf

```diff
@@ -54,20 +54,26 @@ variable "domain" {
 
 variable "db_allocated_storage" {
   description = "Allocated storage for database instance in GiB."
-  type        = number
   default     = 1000
+  type        = number
 }
 
 variable "db_instance_class" {
   description = "Instance class of the RDS instance."
-  type        = string
   default     = "db.t3.micro"
+  type        = string
 }
 
 variable "db_iops" {
   description = "The requested number of I/O operations per second that the DB instance can support."
-  type        = number
   default     = 1000
+  type        = number
+}
+
+variable "dataset_url" {
+  description = "URL of the dataset to restore in the Bamboo instance"
+  type        = string
+  default     = "https://centaurus-datasets.s3.amazonaws.com/bamboo/dcapt-bamboo.zip"
```

Comment:
I am thinking to make it optional. If we can find a way to default this to null or empty string and in case the user does not change it in initial configuration, then we can ignore the data import and otherwise  perform `dataset.tf` and import it in `helm_release.tf`.

Reply:
You are right, yesterday after testing it was clear we need to make it optional as it takes ~30 mins to restore this dataset. I have done the change.